ingress/controllers/gce/README.md

```diff
@@ -381,22 +382,7 @@ metadata:
 type: Opaque
 ```
 
-Referencing this secret in an Ingress will tell the Ingress controller to secure the channel from the client to the loadbalancer using TLS:
-
-```yaml
-apiVersion: extensions/v1beta1
-kind: Ingress
-metadata:
-  name: no-rules-map
-spec:
-  tls:
-  - secretName: testsecret
-  backend:
-    serviceName: s1
-    servicePort: 80
-```
-
-This creates 2 GCE forwarding rules that use a single static ip. Both `:80` and `:443` will direct traffic to your backend, which serves HTTP requests on the target port mentioned in the Service associated with the Ingress.
+Referencing this secret in an Ingress will tell the Ingress controller to secure the channel from the client to the loadbalancer using TLS.
```

Comment:
> Is the ingress controller hardcoded to pull the tls.crt and tls.key subfields or is this for the example at hand ?

Yeah, it's mentioned above: "The TLS secret must contain keys named `tls.crt` and `tls.key` that contain the certificate and private key to use for TLS, eg:". It needs to match: https://github.com/kubernetes/kubernetes/blob/master/pkg/api/types.go#L2696 (linked to that page). 

> Is there any logic that validates the TLS cert and private key are valid and not expired (in the case of the cert) when the yaml create is posted ?

That'll show up as an event at runtime. kubectl create secret will do some validation. 

> Would be good to spell out how to debug TLS failures - point them to a good FAQ.

I'll send a follow up. This part is just like they'd debug TLS failures outside GCE.